repository-data/webfiles/src/main/resources/site/freemarker/common/macro/sectionNav.ftl

```diff
@@ -0,0 +1,17 @@
+<#ftl output_format="HTML">
+
+<#macro sectionNav links = [{ "url": "#section-anchor", "title": "Title"  }] title="Page contents">
```

Comment:
I'd consider not giving these default values as you would never want to fall back on them in real life

Reply:
Makes sense, I'll remove them